doc/fork_support.md

```diff
@@ -0,0 +1,49 @@
+# Background #
+
+In Python multithreading is ineffective at concurrency for CPU bound tasks
+due to the GIL.
```

Comment:
nit: weird line break here, and there shouldn't be a period after both `GIL.` and after the parenthetical `(global interpreter lock).`

Reply:
Fixed.